### PR TITLE
Categorical Hinge Loss Doc Updated

### PR DESCRIPTION
**Pull request by ayushmankumar7**
Issue : #36807  @pavithrasv Check this

### CODE REVIEW

File: tensorflow/python/keras/losses.py [hunk 1]
```diff
@@ -861,7 +861,7 @@ class CategoricalHinge(LossFunctionWrapper):
   """Computes the categorical hinge loss between `y_true` and `y_pred`.
 
   `loss = maximum(neg - pos + 1, 0)`
-  where `neg = sum(y_true * y_pred)` and `pos = maximum(1 - y_true)`
+  where `neg=maximum((1-y_true)*y_pred) and pos=sum(y_true*y_pred)`
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: Thank you, can you make the same change in the `categorical_hinge` function below?

Comment (ayushmankumar7) [reply tensorflow/python/keras/losses.py]: Sure

Comment (ayushmankumar7) [reply tensorflow/python/keras/losses.py]: Done.

**Comment by ayushmankumar7** [comment]
I updated it.

**Review by pavithrasv — APPROVED** [review]

**Comment by ayushmankumar7** [comment]
Thanks for the approval
